planet/clients/subscriptions.py

```diff
@@ -89,7 +91,11 @@ class _SubscriptionsPager(Paged):
             """Navigates pages of messages about subscriptions."""
             ITEMS_KEY = 'subscriptions'
 
-        params = {'status': [val for val in status or {}]}
+        params = {
+            'status': [val for val in status or {}]
+        }  # type: Dict[str, Union[str, Sequence[str]]]
+        if source_type is not None:
+            params['source_type'] = source_type
```

Comment:
I can't seem to dig myself out of the typing on this one:

- With no typehint, mypy fails with `error: Incompatible types in assignment (expression has type "str", target has type "list[str]"), because of the initialization for status
- With type hints and import of Dict and Union, mypy passes but yapf fails with a linting error because the imported "aren't used" (since the hint is a comment)
- And removing the Dict and Union gets us back to a mypy failure because Dict and Union aren't imported

Anyone who's a bit more familiar with this toolset have a solution?

Reply:
I've currently got it working with `#type: ignore`, which isn't as clean as I'd like, but is the only solution I can figure out.

Reply:
Try adding the import statement at the top of the file: 

```python
from typing import Dict, Union, Sequence
```

And defining the type hints like this:

```python
params: Dict[str, Union[str, Sequence[str]]] = {
    'status': [val for val in status or {}]
}
if source_type is not None:
    params['source_type'] = source_type
```

Reply:
That appears to have worked. Thanks!